wagtail/wagtailcore/tests/test_fields.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+from wagtail.wagtailcore.fields import RichTextField
+
+
+class TestRichTextField(TestCase):
+
+    def test_has_field_options_attribute(self):
+        field = RichTextField()
+        self.assertTrue(hasattr(field, 'field_options'))
+
+    def test_field_options_widget_default(self):
```

Comment:
We can remove `test_has_field_options_attribute`,  `test_field_options_widget_default` and `test_formfield_has_widget`, because we don't need to care about internal implementation ;)